ouroboros-network/src/Ouroboros/Network/Socket.hs

```diff
@@ -146,6 +146,32 @@ hexDump :: BL.ByteString -> String -> IO ()
 hexDump buf out | BL.empty == buf = say out
 hexDump buf out = hexDump (BL.tail buf) (out ++ printf "0x%02x " (BL.head buf))
 
+
+-- | A channel which wraps each message as an 'MuxSDU', each sdu is send as
+-- 'Mx.Muxcontrol'.
+--
+muxBearerAsControlChannel
```

Comment:
This function seems out of place in `Socet.hs`. It should be in `Mux.hs` so that it is available for, pipes, windows named pipes (in the future), and MuxSTM queues.

Reply:
Fixed.